### ISSUE TITLE
Testimonials on pricing page?

### ISSUE THREAD

**Issue by arikfr** [getredash/website#141]
Like this:

![image](https://user-images.githubusercontent.com/71468/49422840-083e6380-f79e-11e8-8fbc-391c3818cc9e.png)

It's best to see the real thing to understand the interaction. It's from [Netlify's pricing page](https://www.netlify.com/pricing/).

@kocsmy what do you think?

**Comment by kocsmy** [comment]
Yep, I think it's a good idea to add some testimonials there but we could use the same boxes we're already using on Customers page.
